docs/gettingstarted.rst

```diff
@@ -41,24 +41,66 @@ Quickstart
 
      $ make dockersetup
 
+
+   You can run ``make dockersetup`` any time you want to wipe the database, pick
+   up changes in static data, stored procedures, types, migrations, etc.
+
    .. Warning::
 
-      This is a work in progress, isn't idempotent, and is fussy about the state
-      of things.
+      This is a work in progress and might be fussy about some things.
 
       Pull requests welcome!
 
 
-At that point, you should have a basic functional Socorro development
+At this point, you should have a basic functional Socorro development
 environment.
 
 See :ref:`webapp-chapter` for additional setup and running the webapp.
 
 See :ref:`processor-chapter` for additional setup and running the processor.
 
 
+Updating data in a dev environment
+==================================
+
+Wiping the db
+-------------
+
+Any time you want to wipe the database and start over, run ``make dockersetup``.
+
+
+Updating release data
+---------------------
+
+Data on releases comes from running ftpscraper. After you run ftpscraper, you
+have to run featured-versions-automatic which will update the featured versions
+list.
+
+We put all that in a single shell script.
+
+Run::
+
+  $ docker-compose run processor docker/run_update_release_data.sh
+
+
+.. Warning::
+
+   This can take a long while to run and if you're running it against an
+   existing database, then ftpscraper will "catch up" since the last time it ran
+   which can take a long time, maybe hours.
+
+
+General architecture
+====================
+
+.. image:: block-diagram.png
+
+Arrows direction represents the flow of interesting information (crashes,
+authentication assertions, cached values), not trivia like acks.
```

Comment:
Yeah. I want to switch this to dot and graphviz, but haven't, yet. I'm using that with Antenna and it's been good:

http://antenna.readthedocs.io/en/latest/architecture.html#high-level-architecture

It has the advantage of being text-based and easier to edit in the docs:

https://github.com/mozilla-services/antenna/blob/master/docs/architecture.rst#high-level-architecture

I think that's outside the scope of this pass, though.